### ISSUE TITLE
Present the UI within the Electron app view

### ISSUE THREAD

**Issue by DrewLX** [bitfocus/companion#223]
**Describe the feature**
I would prefer it if the control UI didn't require a browser window, but rather displayed that within the Electron app window that already exists. This could be easily implemented - even as an iFrame if you wanted to be super lazy.

**Usecases**
Many of my show machines I wouldn't have a browser running, so it's a bit cleaner to my mind, and I know a few other Companion users that have made similar comments.

**Comment by DrewLX** [comment]
I'm not talking about removing any features - I like the ability to do it, just not that its the only way to do it. Its a very small easy feature to implement and makes it a simpler and easier program to use.

**Comment by willosof** [comment]
I’ll think about it :)

**Comment by willosof** [comment]
Unfortuneately we need to prioritize other more important issues before we can look into this. But you're welcome to do the changes in the code and make a pull request :)

**Comment by Julusian** [comment]
I did look at this and think 'this sounds pretty easy to do', but then realised there is more to it than I first thought.

Opening it as a separate window would be easy to do, but where should that be done from? I can't figure out where to put a button that wont cause confusion or make the launcher feel cluttered.

Do we need to be careful about navigation inside of this window? I expect it wont have a back button or other typical browser things.

In conclusion, I don't think this will work very well without causing confusion so I am thinking it shouldnt be done. If someone has a good plan on how it could be done, please provide some screenshots or a PR
